app/Models/Client.php

```diff
@@ -39,6 +63,13 @@ public function leads()
             ->orderBy('created_at', 'desc');
     }
 
+    public function contacts()
```

Comment:
This function sorts, I don't think the sorting should be done there but where it's used instead, but a small thing.

Reply:
Sorting seemed to be the norm in the other relationships that already existed, so I honored that in my contacts relationship as well. I think I just copied the tasks relationship and changed the class it used (from Tasks::class to Contacts::class).

Reply:
Fair enough :) No problems with this, yes there are some questionable choices in the code :)